src/ui/profile_preferences.ui

```diff
@@ -42,7 +42,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>8</number>
```

Comment:
:bulb: Please ensure you are showing the ***first*** tab when you save from the Qt Designer form editor - whilst there is hard code to ensure the first tab is always selected when **this** form/dialogue is opened in the `dlgProfilePreferences` constructor, not doing so introduces this pointless change to the source code...